cfpb/consumerfinance.gov
# Move draftail dependencies to cfpb-admin assets app

~~These dependencies appear unused in the project. I may be missing how these are used, so please review draftail functionality closely.~~

They are used to satisfy peerDependencies of the draftail plugin.

## Removals

- ~~Remove unused draftail dependencies.~~

Edit:
## Changes

- Move draftail dependencies from global package.json to a new cfpb-admin assets app.
- Remove dedicated scripts:admin gulp task and use existing assets app infrastructure.

## How to test this PR

1. PR checks should pass.
2. Draftail should work as before. I'm unclear how to test… maybe 

You can checkout this branch locally and use ./runserver.sh
or use the Docker container that this Pull Request creates to:

1. Open or create a wagtail page
2. Modify the rich text
3. Click to save the changes
4. Verify the changes are correctly saved with no errors locked

From https://github.com/cfpb/consumerfinance.gov/pull/5899
